.github/workflows/quality.yml

```diff
@@ -12,8 +12,18 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
 
-      - name: Build Docker image
-        run: docker build -t grafana-sync-action .
+      - name: Install dependencies
+        env:
+          CI: true
```

Comment:
GitHub Actions sets this to `true` by default: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables